changelogs/changelog.yaml

```diff
@@ -1529,3 +1529,41 @@ releases:
         name: ome_alert_policies
         namespace: ''
     release_date: '2023-09-29'
+  8.4.0:
+    changes:
+      bugfixes:
+        - idrac_server_config_profile - The import scp module fails to appropriately
+          handle the absence of a file, incorrectly reporting success instead of the
+          expected failure.
+          (https://github.com/dell/dellemc-openmanage-ansible-modules/issues/544).
+        - idrac_firmware - Issue(276335) - This module fails on the Python 3.11.x
+          version with NFS shares. Use a different Python version or Share type.
+      known_issues:
+        - ome_device_quick_deploy - Issue(275231) - This module does not deploy a
+          new configuration to a slot that has disabled IPv6.
+        - ca_path missing - Issue(275740) - The roles idrac_attributes, redfish_storage_volume,
+          and idrac_server_powerstate have a missing parameter ca_path.
+        - ome_device_network_services - Issue(212681) - The module does not provide
+          a proper error message if unsupported values are provided for the following
+          parameters- port_number, community_name, max_sessions, max_auth_retries, and
+          idle_timeout.
+        - ome_device_power_settings - Issue(212679) - The module displays the following
+          message if the value provided for the parameter ``power_cap`` is not within
+          the supported range of 0 to 32767, ``Unable to complete the request because
+          PowerCap does not exist or is not applicable for the resource URI.``
+        - ome_smart_fabric_uplink - Issue(186024) - Despite the module supported by
+          OpenManage Enterprise Modular, it does not allow the creation of multiple
+          uplinks of the same name. If an uplink is created using the same name as an
+          existing uplink, the existing uplink is modified.
+        - idrac_firmware - Issue(279282) - Unable to update the firmware using HTTP,
```

Comment:
idrac_firmware - Issue(279282) - This module does not support firmware update using HTTP, HTTPS, and FTP shares with Authentication on iDRAC8.

Reply:
Yes updated